docs/sources/datasources/grafana-pyroscope.md

```diff
@@ -61,26 +63,27 @@ The query editor gives you access to a profile type selector, a label selector,
 
 To access the query editor:
 
-1. Sign into Grafana or Grafana Cloud.
-1. Select your Pyroscope data source.
-1. From the menu, choose **Explore**.
+1.  Sign into Grafana or Grafana Cloud.
+1.  Select your Pyroscope data source.
+1.  From the menu, choose **Explore**.
 
-1. Select a profile type from the drop-down menu.
+1.  Select a profile type from the drop-down menu.
 
     {{< figure src="/media/docs/pyroscope/query-editor/select-profile.png" class="docs-image--no-shadow" max-width="450px" caption="Profile selector" >}}
 
-1. Use the labels selector input to filter by labels. Pyroscope uses similar syntax to Prometheus to filter labels.
-Refer to [Pyroscope documentation](https://grafana.com/docs/pyroscope/latest/) for available operators and syntax.
+1.  Use the labels selector input to filter by labels. Pyroscope uses similar syntax to Prometheus to filter labels.
+    Refer to [Pyroscope documentation](https://grafana.com/docs/pyroscope/latest/) for available operators and syntax.
+
+        While the label selector can be left empty to query all profiles without filtering by labels, the profile type or app must be selected for the query to be valid.
 
-    While the label selector can be left empty to query all profiles without filtering by labels, the profile type or app must be selected for the query to be valid.
+        Grafana doesn't show any data if the profile type or app isn’t selected when a query runs.
```

Comment:
With this level of indentation, this section is rendered as a code block.

Reply:
it's weird because it looked okay in the preview and didn't appear as a code block. I should try it again and verify. 
Happy to fix it.

Reply:
<img width="1624" alt="Screenshot 2024-01-25 at 21 24 11" src="https://github.com/grafana/grafana/assets/4599384/19d04b72-17d0-4f6b-86f1-0947e3ee6a20">

Probably changed as a result of running `prettier` when the file was in an ambiguous form. 

I'd expect the indentation of the paragraphs to match the indentation of the first character of the list item.

For example:

```markdown
1. List item.

   Paragraph as part of the first list item.
```

Reply:
I think you're right that it was Prettier that added the spaces. They should be fixed now. I removed the extra spaces and put things back in line.